products/jse_sub/behat.yml

```diff
@@ -0,0 +1,15 @@
+imports:
```

Comment:
I created this for the PICAL sites you were talking about. I believe you wanted another product directory for it? Correct me if I'm wrong